<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>822</width>
    <height>447</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLCDNumber" name="hourLCD">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>131</width>
      <height>121</height>
     </rect>
    </property>
    <property name="digitCount">
     <number>2</number>
    </property>
    <property name="value" stdset="0">
     <double>22.000000000000000</double>
    </property>
    <property name="intValue" stdset="0">
     <number>22</number>
    </property>
   </widget>
   <widget class="QLCDNumber" name="minuteLCD">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>10</y>
      <width>131</width>
      <height>121</height>
     </rect>
    </property>
    <property name="digitCount">
     <number>2</number>
    </property>
    <property name="value" stdset="0">
     <double>22.000000000000000</double>
    </property>
    <property name="intValue" stdset="0">
     <number>22</number>
    </property>
   </widget>
   <widget class="QLCDNumber" name="secondLCD">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>10</y>
      <width>131</width>
      <height>121</height>
     </rect>
    </property>
    <property name="digitCount">
     <number>2</number>
    </property>
    <property name="value" stdset="0">
     <double>22.000000000000000</double>
    </property>
    <property name="intValue" stdset="0">
     <number>22</number>
    </property>
   </widget>
   <widget class="QPushButton" name="imageDownloadButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Image Download/Display</string>
    </property>
   </widget>
   <widget class="QLabel" name="imageLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>190</y>
      <width>411</width>
      <height>211</height>
     </rect>
    </property>
    <property name="text">
     <string>ImageLabel</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="zipCodeEdit">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>140</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>98119</string>
    </property>
   </widget>
   <widget class="QPushButton" name="weatherDownloadButton">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>140</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Weather Download</string>
    </property>
   </widget>
   <widget class="QLabel" name="weatherLabel">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>190</y>
      <width>231</width>
      <height>141</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>822</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
